<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>881</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1460</width>
    <height>881</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1460</width>
    <height>881</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>1240</width>
     <height>840</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1500</width>
     <height>840</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>810</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>810</height>
         </size>
        </property>
        <property name="title">
         <string>SQL</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edt_sql_port">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>120</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="edt_sql_user">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>150</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="edt_sql_pw">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>180</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>351</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Please specify your MySQL credentials below</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_dl_db">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>120</y>
           <width>141</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Download
database</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QTabWidget" name="tabs_sql">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>461</width>
           <height>591</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QTableWidget" name="table_sql1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>431</width>
             <height>541</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <widget class="QTableWidget" name="table_sql2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>431</width>
             <height>541</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="btn_sql_select">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>441</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select .sql file</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="2">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="group_km">
        <property name="title">
         <string>k-means</string>
        </property>
        <widget class="QSpinBox" name="spin_km_k">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>14</number>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_km_run">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>20</y>
           <width>91</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>k:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_km_plot">
         <property name="geometry">
          <rect>
           <x>720</x>
           <y>20</y>
           <width>91</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
        <widget class="QComboBox" name="combo_km1">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>30</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="combo_km2">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>31</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QRadioButton" name="radio_km_forgy">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Forgy</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="radio_km_random">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Random</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="radio_km_euclyd">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>60</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Euclydian</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
        <widget class="QTableWidget" name="table_km">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>931</width>
           <height>271</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_km_plot_all">
         <property name="geometry">
          <rect>
           <x>830</x>
           <y>20</y>
           <width>91</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Plot all</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Final centroid coordinates and classification</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" colspan="2">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QGroupBox" name="group_knn">
        <property name="title">
         <string>k-nearest-neighbours</string>
        </property>
        <widget class="QComboBox" name="combo_knn">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>40</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_knn_run">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>30</y>
           <width>91</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spin_knn_k">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>40</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>106</x>
           <y>40</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>k:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_knn_plot">
         <property name="geometry">
          <rect>
           <x>750</x>
           <y>30</y>
           <width>91</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="table_knn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>931</width>
           <height>271</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>50</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Plot object:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>271</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Test set classification and coordinates</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1460</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
